src/text/text2.asm

```diff
@@ -1715,67 +1715,67 @@ Text023e: ; 3bbf8 (e:7bf8)
 	text "No cards chosen."
 	done
 
-Text023f: ; 3bc0a (e:7c0a)
+YourPokemonText: ; 3bc0a (e:7c0a)
 	text "Your Pokémon"
 	done
 
-Text0240: ; 3bc18 (e:7c18)
+YourDiscardPileText2: ; 3bc18 (e:7c18)
 	text "Your Discard Pile"
 	done
 
-Text0241: ; 3bc2b (e:7c2b)
+YourHandText: ; 3bc2b (e:7c2b)
 	text "Your Hand"
 	done
 
 Text0242: ; 3bc36 (e:7c36)
 	text "To Your Play Area"
 	done
 
-Text0243: ; 3bc49 (e:7c49)
+OpponentsPokemonText: ; 3bc49 (e:7c49)
 	text "Opponent's Pokémon"
 	done
 
-Text0244: ; 3bc5d (e:7c5d)
+OpponentsDiscardPileText2: ; 3bc5d (e:7c5d)
 	text "Opponent's Discard Pile"
 	done
 
-Text0245: ; 3bc76 (e:7c76)
+OpponentsHandText: ; 3bc76 (e:7c76)
 	text "Opponent Hand"
 	done
 
 Text0246: ; 3bc85 (e:7c85)
 	text "To Opponent's Play Area"
 	done
 
-Text0247: ; 3bc9e (e:7c9e)
+PlayersPlayAreaText: ; 3bc9e (e:7c9e)
```

Comment:
I prefer _Duelist_ for ``<RAMNAME>`` texts, as in ``DuelistPlayAreaText``, as this usually references the turn holder's name. I would use _Player_ if the text specifically said "Player" or the player's name.